server/container_update_resources.go

```diff
@@ -27,13 +27,21 @@ func (s *Server) UpdateContainerResources(ctx context.Context, req *types.Update
 	}
 
 	if req.Linux != nil {
-		resources := toOCIResources(req.Linux)
+		updated, err := s.nri.updateContainer(ctx, c, req.Linux)
+		if err != nil {
+			return nil, err
```

Comment:
Same here. For most calls you choose to just warn, but in here you fail. Why is that?

Reply:
It's a potentially state-altering call. We treat failure in those as a failure in the triggering CRI call, propagating the error all the way back.